content/docs/command-reference/import.md

```diff
@@ -103,6 +103,11 @@ repo at `url`) are not supported.
   > [Importing and updating fixed revisions](#example-importing-and-updating-fixed-revisions)
   > example below).
 
+- `--no-exec` - create `.dvc` file without actually downloading `url`. E.g. if
+  the file or directory already exist it can be used to skip download.
+  `dvc commit <out>.dvc` should be used to calculate the URL and data hash,
```

Comment:
@BurnzZ not sure about the `calculate URL` part, could you clarify what's happening here?

Reply:
It's a copy/paste of the homologue option in https://dvc.org/doc/command-reference/import-url#options

Reply:
I.e. yes, that part about calculating hashes doesn't apply to DVC repo imports, we can just remove it as no other similar process occurs I think.

Reply:
It should be `existS` and probably `THE download` or `downloadING`.

`E.g. if the file or directory already exist it can be used to skip download.` I'm not sure I understand this scenario? 😄 Do we run `dvc import --no-exec` on top of an existing directory - what is the goal? I thought, the primary use case is to postpone the download because we need a partial run (that is supported in `pull` but not `import`) OR the download needs to be run in another machine (we just cook metadata).

Reply:
@jorgeorpinel to my mind `--no-exec` in the import-url has to be updated as well. `Calculate URL and data hash` part is not clear to me.

@dmpetrov it is the same as `import-url --no-exec`. Scenario is - create .dvc but don't download (if data exists already). E.g. you did `wget something` and now want to have a .dvc files to track it, but don't want to download data again.

Reply:
`the download needs to be run in another machine (we just cook metadata).` - this scenario fits as well, I guess

`we just cook metadata` - is a good description I think.

Reply:
Hi everyone! Thanks for the feedback!

> It's a copy/paste of the same option in https://dvc.org/doc/command-reference/import-url#options.
@jorgeorpinel That's right! I've also updated the docs for `import-url` as per @shcheklein's reminder.

I've refined the `--no-exec` definition as per your feedback.